src/api/resources/classification/image.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import Q
+from rest_condition import And
+from rest_framework import status
+from rest_framework.response import Response
+
+from ...exceptions import check_uuid
+from ...models import BENTHICPQT_PROTOCOL, CollectRecord, Image, ObsBenthicPhotoQuadrat
+from ...permissions import CollectRecordOwner
+from ..base import BaseAPISerializer, BaseProjectApiViewSet
+
+
+class ImageSerializer(BaseAPISerializer):
```

Comment:
Not sure what the answer is but noting our discussion of what read-only nested info might want to be included here: classification status, list of points with annotations per point, etc.